### DISCUSSION TITLE
Try to upload Resources error mkdir /files: permission denied

### DISCUSSION THREAD

**Discussion by Vladimir-Va** [casdoor/casdoor#1380] [Q&A]
Hello how to correct setup Local Storage Providers?
I try to set Domain name as:
http://mydomainename.com/images
https://mydomainename.com/images
http://ipadress/images
and many other variations
Same result.
And if i go to hiperlink in Applications/Providers i redirect on 404 page.

**Comment by hsluoyz** [comment]
@Vladimir-Va issue created at: https://github.com/casdoor/casdoor/issues/1382
